### ISSUE TITLE
.Net wrapper libraries ...?

### ISSUE THREAD

**Issue by polterguy** [official-stockfish/Stockfish#1523]
The header says it all. I want to wrap StockFish in a webapp frontend GUI, built in [Phosphorus Five](https://github.com/polterguy/phosphorusfive). But I'll need to be able to either have som .Net API, or create my own somehow ...

**Comment by Rocky640** [comment]
Here is one under the MIT licence
https://github.com/manixaist/AllPawnsMustDie

**Comment by polterguy** [comment]
Thx, I was kind of hoping for a library, but I assume I can use the above example to create one myself.

**Comment by snicolet** [comment]
@polterguy Any update on your project? Do you have something working now? :-)

**Comment by polterguy** [comment]
Sorry, I had to put this one on the back-burner for a while, since I've got some other things I am tending to at the moment, creating a new release for the main Phosphorus Five system. However, this is one thing I would _really_ enjoy doing - But since the prospects of earning some money on this seems far fetched, I must prioritise getting food on my table first ... :)

However, when I've wrapped up the new release, this is _definitely_ one of those things I would seriously enjoy doing, being a hobby player myself - Arguably giving small chess clubs and such a way to easily create their own websites (ALA; chess.com) to have their own infrastructure similar to chess.com - Although obviously, I don't have the resources to bring it as far as chess.com and other commercial sites have created their systems. Still, I think this could be a killer _"website addon"_ for smaller chess clubs and such, wanting to follow up on their own members, slightly more _"privately"_, than by using a massive online website for playing - Among other things ...

Features I am considering is keeping records of all matches, analysing matches, keeping a player's score, multiuser (obviously), playing against other players, and finally (of course), integrating StockFish, not only as a bot player, but also to analyse human2human matches.

Not sure if I want to bring it very much further, such as creating chat interfaces, badges, etc, etc, etc - Since this could also easily be done with other systems from before - But I guess we'll see with time ...

Keep me on my toes if you don't hear from me in a couple of months ... :)
